core/src/main/scala/kafka/tools/StreamsResetter.java

```diff
@@ -246,24 +329,36 @@ private boolean isIntermediateTopic(final String topic) {
         return options.valuesOf(intermediateTopicsOption).contains(topic);
     }
 
-    private void deleteInternalTopics(final ZkUtils zkUtils) {
-        System.out.println("Deleting all internal/auto-created topics for application " + options.valueOf(applicationIdOption));
+    private void maybedeleteInternalTopics(final ZkUtils zkUtils) {
+        Boolean dryRun = options.has(dryRunOption);
+
+        if (!dryRun) {
+            System.out.println("Deleting all internal/auto-created topics for application " + options.valueOf(applicationIdOption));
+        } else {
+            System.out.println("Following internal/auto-created topics will be Deleted for application " + options.valueOf(applicationIdOption));
```

Comment:
Agreed. Deleted the condition, as anyway there is this displayed on start, when run with dry-run option.
----Dry run displays the actions which will be performed when running Streams Reset Tool----